pkg/wsman/cim/power/constants.go

```diff
@@ -56,3 +56,36 @@ const (
 	// Power Cycle (Off - Hard Graceful)
 	PowerCycleOffHardGraceful PowerState = 16 // ?
 )
+
+const (
+	EnabledStateUnknown EnabledState = iota
```

Comment:
Lets remove EnabledState and RequestedState constants.  This Class doesn't have a put call where these would be used.